### ISSUE TITLE
EF.Reverse.POCO.v3.ttinclude different inside the distribution zip

### ISSUE THREAD

**Issue by clcdrn** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#808]
Download [https://github.com/sjh37/EntityFramework-Reverse-POCO-Code-First-Generator/archive/refs/tags/v3.8.4.zip](https://github.com/sjh37/EntityFramework-Reverse-POCO-Code-First-Generator/archive/refs/tags/v3.8.4.zip)

1) Shouldn't 
EntityFramework-Reverse-POCO-Code-First-Generator-3.8.4.zip\EntityFramework-Reverse-POCO-Code-First-Generator-3.8.4\EntityFramework.Reverse.POCO.Generator\EF.Reverse.POCO.v3.ttinclude
be the same as
EntityFramework-Reverse-POCO-Code-First-Generator-3.8.4.zip\EntityFramework-Reverse-POCO-Code-First-Generator-3.8.4\EntityFramework Reverse POCO Generator\ItemTemplates\efrpoco.zip\EF.Reverse.POCO.v3.ttinclude
except for the CR+LF line endings?

2) Also, what about the EF6.ttinclude?

Maybe it doesn't matter, but the question is, which is correct?

**Comment by sjh37** [comment]
Hi. The correct one is without EF6.ttinclude. I was going to include it but then Microsoft released a fix for it so no need for it.

**Comment by clcdrn** [comment]
What about the first question? Sorry, maybe I should have opened a separate ticket as the questions they are unrelated.

**Comment by sjh37** [comment]
Yes they should be the same! Thanks for letting me know. I shall investigate. However it will be next week before I can look at it.

**Comment by sjh37** [comment]
In v4 the item template zip is built by BuildTT directly from the checked-in `EF.Reverse.POCO.v4.ttinclude` at release time, so the two copies can no longer drift apart. `EF6.ttinclude` is deliberately not shipped, as discussed above. Closing.
